scripts/mirror.js

```diff
@@ -56,37 +56,45 @@ const { argv } = require('yargs').command(
 );
 
 /**
- * @param {string} dest_browser
- * @param {ReleaseStatement} source_release
+ * @param {string} destBrowser
+ * @param {ReleaseStatement} sourceRelease
+ * @param {string} sourceVersion
  * @returns {ReleaseStatement|boolean}
  */
-const getMatchingBrowserVersion = (dest_browser, source_release) => {
-  const browserData = browsers[dest_browser];
+const getMatchingBrowserVersion = (
+  destBrowser,
+  sourceBrowser,
+  sourceVersion,
```

Comment:
Maybe it wouldn't hurt here either to use an object parameter `source: { browser, version }` instead?